ui/components/app/assets/token-cell/token-cell.tsx

```diff
@@ -17,10 +30,12 @@ export default function TokenCell({ address, image, symbol, string, onClick }) {
   );
   const title = tokenData?.name || symbol;
   const tokenImage = tokenData?.iconUrl || image;
-  const formattedFiat = useTokenFiatAmount(address, string, symbol);
+  const formattedFiat = useTokenFiatAmount(address, string, symbol, {}, false);
   const locale = useSelector(getIntlLocale);
   const primary = new Intl.NumberFormat(locale, {
     minimumSignificantDigits: 1,
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
```

Comment:
I sadly see no easy way around this one, as changing this broke things way downstream. I dub this variable "the string who must not be named" from here on out

Reply:
lol a variable named 'string' that actually represents the balance of the token.

It's a little weird that we're calling `.toString` on something already typed as string.  If we don't know why, we can leave it.

I also tried to make things \`${number}\` (a string with numeric contents) instead of plain string, but couldn't reach all places